packages/trimerge-sync/src/TrimergeClient_2Users.test.ts

```diff
@@ -101,20 +100,20 @@ describe('TrimergeClient: 2 users', () => {
     expect(basicGraph(store, client)).toMatchInlineSnapshot(`
 Array [
   Object {
-    "graph": "undefined -> DuQe--Vh",
-    "step": "User a: initialize",
+    "graph": "undefined -> Zob0dMmD",
```

Comment:
is the fact that we're using editMetadata to compute hash going to cause problems? if this has local store and client id in it then won't we never generate the same refs?

Reply:
I tried to remove metadata from compute hash in this PR, did you see any places I missed?